content/slice/language-guide/class-types.md

```diff
@@ -44,6 +44,7 @@ example:
 ```slice
 class CarPart {
     id: string
+    revision: int32
 }
```

Comment:
I'm fine with adding this field to keep the `CarPart` structs consistent, but this section needs to be read through and fixed.

Just a couple lines below here, we say that `RearBumper` has 2 fields `id` and `color`.
But now it has 3 fields, since `revision` now exists. There may be other things in need of fixing now too.

Reply:
Fixed